trace/exporterutil/examples_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package exporterutil_test
+
+import (
+	"fmt"
+
+	"go.opencensus.io/trace"
+	"go.opencensus.io/trace/exporterutil"
+	"golang.org/x/net/context"
+)
+
+type testExporter struct {
+	buf string
+}
+
+func (e *testExporter) ExportSpan(spanData *trace.SpanData) {
+	e.buf += spanData.Name + "\n"
+}
+
+func (e *testExporter) Flush() {
+	fmt.Print(e.buf)
+}
+
+func ExampleExcludingSpans() {
+	filtered := exporterutil.ExcludingSpans(new(testExporter), exporterutil.WithNamePrefix("exclude-"))
+	trace.RegisterExporter(filtered)
+
+	// sampled and exported
```

Comment:
In Go, comments are proper sentences if they are in their own block.

// Sampled and exported span:

Reply:
the style guide has some contradictory examples: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
I don't really care, just saying it doesn't seem to be a general rule or maybe we should fix the style guide?